app/controllers/distributions_controller.rb

```diff
@@ -44,31 +42,25 @@ def index
 
   def create
     @distribution = Distribution.new(distribution_params.merge(organization: current_organization))
+    @storage_locations = current_organization.storage_locations
```

Comment:
Why was this needed?

Reply:
Looks like it's not!

Reply:
That might have been vestigial. Weird that Rubocop didn't pick it up.

Reply:
Oh, actually -- I know why this is there, and it's intentional, though I agree it could be done better.

When the `#create` action fails, it does `render :new` -- and we need this collection so that the page renders correctly.  We've got 3 different branches that lead to `render :new`, though -- and that seems silly. I'll make an issue.

Reply:
#1003